wca-regulations.md

```diff
@@ -269,10 +269,13 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 9p2a) For each round, advancement conditions must be announced before the round starts, and should not be changed after it has begun. Changes must be made at the discretion of the WCA Delegate, who must carefully consider the fairness of the change.
     - 9p3) If a qualifying competitor withdraws from a round, they may be replaced by the best-ranked non-qualifying competitor from the preceding round.
 - 9s) Each round of each event must have a time limit (see [Regulation A1a](regulations:regulation:A1a)).
-- 9t) If any change occurs to the results of a competitor after the competition has ended (e.g. score-taking mistake, retroactive penalty), such that the competitor should not have advanced to the next round (or was not allowed to take part in the round), all results of this competitor for all subsequent rounds must be removed.
+- 9t) If any change occurs to the results of a competitor after the competition has ended (e.g. score-taking mistake, retroactive penalty), such that the competitor should not have advanced to the next round (or was not allowed to take part in the round), all results of this competitor for all subsequent rounds must be removed (see [Regulation 9u](regulations:regulation:9u)).
     - 9t1) If this makes a round no longer compliant with [Regulation 9p1](regulations:regulation:9p1), results of other competitors must not be removed.
     - 9t2) If this makes a round no longer compliant with [Regulation 9m1](regulations:regulation:9m1), [9m2](regulations:regulation:9m2) or [9m3](regulations:regulation:9m3), any subsequent round must not be removed.
     - 9t3) If any change occurs to the result of a competitor, such that another competitor should not have advanced to the next round, the result of the second competitor must not be removed.
+- 9u) A competition is considered to have ended if all the following requirement are meet:
+    - 9u1) All the scheduled rounds in the WCA website are completed.
+    - 9u2) All the listed WCA Delegates permanently leave the venue. Exception: the WCA Delegate may announce the end of the competition before all the listed WCA Delegates have left the venue (e.g. before the awards ceremony).
```

Comment:
This is perhaps a silly corner case, but I periodically host PBQ competitions at my house. If I don't announce the end of the competition, does it technically never end (until I move)?

Consider adding a clause that it ends at some defined time, like an hour after the last official solve or at the end of the calendar day.

How does this interact with multi-location FMC competitions?

Reply:
Thank you for your comments, I will update the proposal soon.

Reply:
> Consider adding a clause that it ends at some defined time, like an hour after the last official solve or at the end of the calendar day.

It was not that easy for me to clearly define the end of the calendar date. What do you think about the changes?